examples/nitclk/multi-device-configured-acquisition-tclk.py

```diff
@@ -0,0 +1,294 @@
+        #// Get the trigger type from the user
+        print("Type the trigger type\n")
+        print("1 - Edge\n")
+        print("2 - Hysteresis\n")
+        print("3 - Digital\n")
+        print("4 - Window\n")
+        print("5 - Immediate\n")
+        print("Option: ")
+        trigger_type = input("Option: ")
+        triggerHoldoff = 0.0
+        triggerDelay = 0.0
+
+        #Assign the default values for the trigger attribute that depends on the trigger type
+        if trigger_type == 1:
+            trigger_edge_result = scope_service.ConfigureTriggerEdge(niscope_types.ConfigureTriggerEdgeRequest(
+            vi = vi,
+            trigger_source = "0",
+            level = 0.0,
+            slope = niscope_types.TriggerSlope.TRIGGER_SLOPE_NISCOPE_VAL_POSITIVE,
+            trigger_coupling = niscope_types.TriggerCoupling.TRIGGER_COUPLING_NISCOPE_VAL_DC,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_edge_result.status)
+        elif trigger_type == 2:
+            trigger_hysteresis_result = scope_service.ConfigureTriggerHysteresis(niscope_types.ConfigureTriggerHysteresisRequest(
+            vi = vi,
+            trigger_source = "0",
+            level = 0.0,
+            hysteresis = 0.1,
+            slope = niscope_types.TriggerSlope.TRIGGER_SLOPE_NISCOPE_VAL_POSITIVE,
+            trigger_coupling = niscope_types.TriggerCoupling.TRIGGER_COUPLING_NISCOPE_VAL_DC,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_hysteresis_result.status)	
+        elif trigger_type == 3:
+            trigger_digital_result = scope_service.ConfigureTriggerDigital(niscope_types.ConfigureTriggerDigitalRequest(
+            vi = vi,
+            trigger_source = "VAL_PFI_1",
+            slope = niscope_types.TriggerSlope.TRIGGER_SLOPE_NISCOPE_VAL_POSITIVE,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_digital_result.status)	
+        elif trigger_type ==  4:
+            trigger_window_result = scope_service.ConfigureTriggerWindow(niscope_types.ConfigureTriggerWindowRequest(
+            vi = vi,
+            trigger_source = "0",
+            low_level = -0.1,
+            high_level = 0.1,
+            window_mode = niscope_types.TriggerWindowMode.TRIGGER_WINDOW_MODE_NISCOPE_VAL_ENTERING_WINDOW,	
+            trigger_coupling = niscope_types.TriggerCoupling.TRIGGER_COUPLING_NISCOPE_VAL_DC,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_window_result.status)
+        elif trigger_type == 5:
+            trigger_immediate_result = scope_service.ConfigureTriggerImmediate(niscope_types.ConfigureTriggerImmediateRequest(
+            vi = vi
+            ))
+            CheckForError(scope_service, vi, trigger_immediate_result.status)
+
```

Comment:
What happens if the user provides an invalid input? Just for a safe check I would add an else case and throw an error.